lms/templates/student_account/login.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="status already-authenticated-msg hidden">
     <% if (context.currentProvider) { %>
         <p class="message-copy">
-            <%- _.sprintf( gettext("You've successfully signed into %(currentProvider)s, but your %(currentProvider)s account isn't linked with an %(platformName)s account. To link your accounts, go to your %(platformName)s account settings."), context ) %>
+            <%- _.sprintf( gettext("You have successfully signed into %(currentProvider)s, but your %(currentProvider)s account is not linked with any %(platformName)s account. To link your accounts, sign in now using your %(platformName)s password."), context ) %>
```

Comment:
I used "this" because, although the user hasn't signed in to an edX account, he _is_ trying to sign in to a specific account. "Any" sounds a little harsh to me. Since the main thing we're trying to avoid is having to switch between a/an in front of "your platform name", what about this?

You have successfully signed into %(currentProvider)s, but your %(currentProvider)s account does not have a linked %(platformName)s account. To link your accounts, sign in now using your %(platformName)s password.